src/components/common/Header/MobileHeader/index.js

```diff
@@ -0,0 +1,69 @@
+import React, { Fragment } from "react";
+import { withStyles } from "@material-ui/styles";
+import Icon from "@material-ui/core/Icon";
+import clsx from "clsx";
+import { connect } from "react-redux";
+import { stylesActions } from "../../../../Redux/actionCreators";
+
+import HeaderActions from "../HeaderActions";
+import NavItem from "../NavItem";
+import { useStyles } from "./styles";
+
+const MobileHeader = ({ classes, data, isLoggedIn, hamburgerMenu, updateHamburgerState }) => {
+  const toggleMobileMenu = () => {
+    updateHamburgerState(!hamburgerMenu);
+  };
+
+  if (!hamburgerMenu) {
+    return (
+      <div className={classes.hamburger} onClick={toggleMobileMenu}>
+        <span />
+        <span />
+        <span />
```

Comment:
What does each `span` represent here? If it is being styled then you could extract them into a specific component which would convey the meaning.

Reply:
span is used to create the hanburger icon for the Menu